raiden/tests/integration/fixtures/backend_tester.py

```diff
@@ -0,0 +1,59 @@
+"""eth-tester specific fixtures"""
+import pytest
+import gevent
+from eth_utils import encode_hex, to_checksum_address, decode_hex, denoms
+from raiden.utils import privatekey_to_address
+
+from raiden_libs.test.fixtures.web3 import *  # noqa
+from raiden_libs.test.fixtures.address import *  # noqa
+
+
+@pytest.fixture
+def fund_accounts(web3, blockchain_type, faucet_address, private_keys, ethereum_tester):
+    [
+        ethereum_tester.add_account(encode_hex(key))
+        for key in private_keys
+        if to_checksum_address(privatekey_to_address(key)) not in ethereum_tester.get_accounts()
+    ]
+    [ethereum_tester.send_transaction({
+        'from': faucet_address,
+        'to': to_checksum_address(privatekey_to_address(key)),
+        'gas': 21000,
+        'value': 1 * denoms.ether
+    }) for key in private_keys]
+
+
+@pytest.fixture
+def spawn_autominer(web3):
```

Comment:
Why is this needed? Is the mining blocking otherwise?

Reply:
The tester only mines a block once you submit a transaction, but no empty blocks are mined. Block generation is required to confirm the transactions.